components/usage-api/usage/v1/billing.proto

```diff
@@ -80,14 +78,16 @@ message CreateStripeCustomerResponse {
 
 message CreateStripeSubscriptionRequest {
   string attribution_id = 1;
-  string stripe_intent_id = 2;
-
-  int32 usage_limit = 3;
+  string setup_intent_id = 2;
+  int64 usage_limit = 3;
 }
 
 message CreateStripeSubscriptionResponse {
-  StripeCustomer customer = 1;
-  v1.CostCenter cost_center = 2;
+  StripeSubscription subscription = 1;
+}
+
+message StripeSubscription {
```

Comment:
I created this subscription object that only has an ID for now. Is there a way to import the type from Stripe API itself?

Reply:
We don't actually want to import the Stripe ID directly, as that would couple us to Stripe on a compilation level. String is fine, it should be opaque anyway.